### PR TITLE
use pkl.tmBundle grammar

### PR DESCRIPTION
**Pull request by stackoverflow**
Unfortunately, TextMate snippets are not supported natively in vscode unless you add an extension. So I kept them here.

### CODE REVIEW

File: src/pkl/index.pkl [hunk 1]
```diff
@@ -16,7 +16,6 @@
 
 output {
   files {
-    ["out/pkl.tmLanguage.json"] = import("pkl.tmLanguage.pkl").output
-    ["out/pkl.snippets.json"] = import("snippets.pkl").output
+    ["out/pkl.snippets.json"] = import("../../textmate-bundle/pkl/snippets.pkl").output
```
Comment (bioball) [inline src/pkl/index.pkl:19]: For consistency sake, how about let's keep both of these here?

Sidenote: wouldn't be a bad idea to turn these into packages.

**Review by bioball — APPROVED** [review]
Some suggestions, but overall LGTM

File: src/pkl/index.pkl (hunk 1 above)
Comment (stackoverflow) [reply src/pkl/index.pkl:19]: You mean generate `pkl.tmLanguage.json` here from the source pkl instead of using the pre-rendered textmate language?

Comment (bioball) [reply src/pkl/index.pkl:19]: Yup

**Pull request merged**